cmd/integration_test/build/buildspecs/vsphere-test-eks-a-cli.yml

```diff
@@ -13,6 +13,7 @@ env:
     T_VSPHERE_TEMPLATE_UBUNTU_1_25: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-25"
     T_VSPHERE_TEMPLATE_UBUNTU_1_26: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-26"
     T_VSPHERE_TEMPLATE_UBUNTU_1_27: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-27"
+    T_VSPHERE_MULTI_TEMPLATE_UBUNTU_1_27: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-27-no-snapshots"
```

Comment:
Why do we need to use this specific one?

Reply:
not specific, just need to use a different one other than `T_VSPHERE_TEMPLATE_UBUNTU_1_27`